spec/core/link.fmf

```diff
@@ -0,0 +1,107 @@
+summary: Link related objects
+
+story:
+    As a user I want to link related tests, stories or external
+    issues so that I can easily find out what the test, plan or
+    story is addressing.
+
+description: |
+    The core attribute ``link`` is used to track relevant objects
+    such as user stories or external issues verified by the given
+    test, source code implementing particular story, blocking
+    issues or parent-child relations.
+
+    The value can be a single link or a list of links. There are
+    two options how to specify the ``target`` of the reference:
+
+    string
+        This can be either a ``name`` of the fmf node within the
+        same metadata tree, a file ``path`` from the tree root to
```

Comment:
Can we have some examples about `name` vs `path`? It looks that name always starts with `/` which path cannot?

Reply:
This should be covered by:

```yaml
link:
  - implemented-by: /tmt/cli.py
  - verified-by: /tests/init/base
  - documented-by: /docs/guide.rst
```

Do you recommend more examples like this?

Reply:
Hm, I didn't realise that. So `name` and `path` cannot be distinguished without checking what is on the disk

Reply:
When implementing the advanced linking (e.g. auto-backward references) or checking backward links in `tmt test lint` I would go with a simple Tree.find(target) check and if found then apply some logic.